IDL/cocovideo.pro

```diff
@@ -12,23 +12,24 @@ PRO COCOVIDEO, coco_datacube, filter, filepath, fps, startstop=startstop, rgbthr
 ;	  COCOPLOT visualisation
 ;
 ; CALLING SEQUENCE:
-;	  Result = COCOVIDEO(coco_data_rgb_int)
+;         COCOVIDEO, coco_datacube, filter, fps, name
 ;
 ; INPUTS:
-;	  coco_datacube:  Input 4D data cube of dimensions [nx, ny, nspect_points, nt]
+;   coco_datacube:  Input 4D data cube of dimensions [nx, ny, nspect_points, nt]
 ;   filter:         Filter of dimensions [nspect_points, 3] 
-;   filepath:       Output path and filename.
 ;   fps:            Frames per second
+;   name:           String containing Ooutput file name.

```

Comment:
Typo in `Ooutput`

Reply:
Done. Also capitalized functions in the code.